### PR TITLE
doc: Case definition setting for 'can have assignee'

### PR DESCRIPTION
**Pull request by maisaritense**
Closes US: https://ritense.tpondemand.com/entity/37537-befe-case-definition-setting-for-can

### CODE REVIEW

File: using-valtimo/case/case.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Case assignee
+
+Case assignee offers a way to configure, per case type, the option to enable set assignee when relevant.
```
Comment (ivo-ritense) [inline using-valtimo/case/case.md:3]: I think this sentence is a bit much. you try to say the same thing twice. I would change it to something like this: 

```suggestion
It is possible to configure per case type if cases of that type can be assigned to users.
```

File: using-valtimo/case/set-case-assignee.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# How to set case assignee to a task
```
Comment (ivo-ritense) [inline using-valtimo/case/set-case-assignee.md:1]:
```suggestion
# How to enable cases to be assignable
```

File: using-valtimo/case/set-case-assignee.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+On the case management page, a setting can be defined to indicate that a case type can have an assignee (checkbox).
```
Comment (ivo-ritense) [inline using-valtimo/case/set-case-assignee.md:3]:
```suggestion
On the case management page, a setting can be changed to indicate that cases of that type can have an assignee (checkbox).
```

File: using-valtimo/case/set-case-assignee.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+It is necessary to access the case details tab to enable the option to add an assignee to a case.
```
Comment (ivo-ritense) [inline using-valtimo/case/set-case-assignee.md:5]:
```suggestion
On this page you can check the Case assignee checkbox to enable assigning cases.
```

**Comment by marijnritense** [comment]
I added BE documentation to this PR so it encompasses the feature as a whole.

**Review by ivo-ritense — APPROVED** [review]

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**